spec/support/reporting_cycle.rb

```diff
@@ -3,14 +3,21 @@ def initialize(activity, financial_quarter, financial_year)
     @activity = activity
     @financial_quarter = financial_quarter
     @financial_year = financial_year
+    @report = nil
   end
 
   def tick
```

Comment:
I'm wondering if there's a better name for this method. It's not obvious to me why calling it means subsequent `history.set_value()` calls are logged against future quarters.

Reply:
I had the same problem and went with `tick` just for the purpose of making progress. What it actually does is simulate the turn-over of the reporting cycle where the current report gets approved, and a new one is opened for the DP to start adding stuff to. Any ideas for better words to use to capture this?